package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "repository": "github:GoogleChrome/puppeteer",
   "engines": {
-    "node": ">=6.4.0"
+    "node": ">=8.9.0"
```

Comment:
As a practical concern, is there anything from 7 that would break us?

Reply:
Not sure. I picked 8 here since it's an LTS release with a clear lifetime.

Reply:
Ok. Turns out this only produces a warning.